src/js/components/App.js

```diff
@@ -1,39 +1,95 @@
 import MenuInput from './MenuInput.js';
 import MenuList from './MenuList.js';
 import Count from './Count.js';
+import { $ } from '../utils/dom.js';
+import { UPDATE_MSG, DELETE_MSG } from '../constants/index.js';
+import { store } from '../store/index.js';
 
 function App() {
-  // NOTE:
-  // enuInfo라는 이름이 적당한지는 모르겠음
-  // 내부에 menu title과 id 값을 가진 object들을 요소로 가짐
-  let menuInfo = [];
   let menuId = 0;
+  let currentCategory = 'espresso';
+  let menu = {
+    espresso: [],
+    frappuccino: [],
+    blended: [],
+    teavana: [],
+    desert: [],
+  };
+
+  const count = new Count();
+  const menuList = new MenuList();
 
-  const setState = updatedMenuList => {
-    menuInfo = updatedMenuList;
+  this.init = () => {
+    if (localStorage.getItem('menu')) menu = store.getItem();
+    this.render();
+    initEventListeners();
   };
 
-  const onUpdate = (id, updatedMenu) => {
-    const updated = menuInfo.map(el => (el.id * 1 === id * 1 ? { ...el, title: updatedMenu } : el));
-    setState(updated);
+  this.render = () => {
+    menuList.render(menu[currentCategory]);
+    count.updateCount({ menuCount: menu[currentCategory].length });
```

Comment:
```suggestion
    const currentMenu = menu[currentCategory];
    menuList.render(currentMenu);
    count.updateCount({ menuCount: currentMenu.length });
```
중복되는 배열 인덱스 호출을 없애고 변수로 선언하는 것도 좋을 것 같아요!